app/components/publications/publication-sidebar.js

```diff
@@ -51,22 +51,17 @@ export default class PublicationsPublicationSidebarComponent extends Component {
 
   @task
   *loadRegulationTypes() {
-    const regulationTypes = yield this.store.query('regulation-type', {
-      sort: 'position',
-    });
-    return regulationTypes;
+    return yield this.store.peekAll('regulation-type').sortBy('position');
```

Comment:
`this.store.peekAll(...)` geeft geen promise terug. De `yield` en "omweg" via de concurrency task zijn dus niet nodig. Deze codelist kan gewoon ingesteld worden in de constructor zoals nu het geval is voor 
```javascript
this.publicationModes = this.store.peekAll('publication-mode').sortBy('position');
```